### PR TITLE
[openhabcloud] Don't follow 302 redirects

### PR DESCRIPTION
**Pull request by digitaldan**
Don't follow 302 redirects (jetty default),  these should be forwarded back to the cloud service.  This is required for authentication to work in OH 3.0

Signed-off-by: Dan Cunningham <dan@digitaldan.com>

### CODE REVIEW

**Review by cpmeister — APPROVED** [review]
LGTM

**Comment by cpmeister** [comment]
The build is complaining about format issues. Try running spotless.

File: bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudService.java [hunk 1]
```diff
@@ -226,7 +226,8 @@ protected void modified(Map<String, ?> config) {
 
         httpClient.setMaxConnectionsPerDestination(DEFAULT_LOCAL_OPENHAB_MAX_CONCURRENT_REQUESTS);
         httpClient.setConnectTimeout(DEFAULT_LOCAL_OPENHAB_REQUEST_TIMEOUT);
-
+        httpClient.setFollowRedirects(false);
+
```
Comment (Hilbrand) [inline bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudService.java:230]:
```suggestion

```

**Review by Hilbrand — APPROVED** [review]
LGTM

**Comment by Hilbrand** [comment]
Your sign off is not valid. It should contain your real name. Shall I put it in when merging this pr?

**Comment by digitaldan** [comment]
Thanks @Hilbrand, I went ahead and changed the comment, let me know if you need me to re-push the commit with an updated sign off.  I had not realized I had been using an incorrect sign off all these years!

**Comment by Hilbrand** [comment]
Ok thanks. It's only because I noticed some recent invalid sign-offs I started looking at a bit closer :smile:

**Comment by digitaldan** [comment]
:-) , Also that errant space was odd too, my local copy does not show it in my editor, but I am using VSCode for the first time to edit Java files, not sure if i have that setup correctly.

**Comment by BClark09** [comment]
Will this also need to be merged into the master branch to see a change in the openHAB 3 snapshots?

**Comment by Hilbrand** [comment]
No it doesn't need to be merged in master. This io addon 2.5.x is automigrated to 3.0 at the moment. So this change will also be available in the openHAB 3. Once we switch to master again all 2.5.x changes are merged to master.

**Comment by kaikreuzer** [comment]
FTR: https://ci.openhab.org/job/openHAB-Addons-port/

**Comment by digitaldan** [comment]
No, the most of the addons are still maintained in 2.5.x and are used when building the 3.x snapshots (with the exception of the zwave binding for the time being)

**Comment by BClark09** [comment]
Righto thanks! Sorry for the noise! :)

**Pull request merged**